sdk/python/tests/compiler/testdata/custom_task_spec.yaml

```diff
@@ -31,21 +31,35 @@ spec:
   pipelineSpec:
     tasks:
     - name: any-name
-      params:
-      - name: other_custom_task_argument_keys
-        value: args
-      taskRef:
+      taskSpec:
         apiVersion: custom_task_api_version
         kind: custom_task_kind
-        name: custom_task_name
+        metadata:
+          annotations:
+            tekton.dev/template: ''
+            valid_container: 'false'
+          labels:
+            pipelines.kubeflow.org/cache_enabled: 'true'
+            pipelines.kubeflow.org/generation: ''
+            pipelines.kubeflow.org/pipelinename: ''
+        spec:
+          raw: raw
+      timeout: 0s
```

Comment:
I don't think custom task can support timeout yet. can we pop this out until we have your Timeout PR merged in?